android/src/main/java/net/no_mad/tts/TextToSpeechModule.java

```diff
@@ -350,6 +350,7 @@ public void onInit(int status) {
                             resolveReadyPromise(p);
```

Comment:
I mean this line here. When we arrive here, these promises have been resolved already, otherwise the call would return at line 339. I think this may be a Copy&Paste mistake taken from the initialisation code, and probably it's not needed here.